bittensor/_axon/__init__.py

```diff
@@ -280,19 +274,19 @@ class AuthInterceptor(grpc.ServerInterceptor):
     def __init__(
         self,
         receiver_hotkey: str,
-        blacklist: Callable = None,
+        metagraph: "bittensor.metagraph",
     ):
         r"""Creates a new server interceptor that authenticates incoming messages from passed arguments.
         Args:
             receiver_hotkey(str):
                 the SS58 address of the hotkey which should be targeted by RPCs
-            black_list (Function, `optional`):
-                black list function that prevents certain pubkeys from sending messages
         """
         super().__init__()
         self.nonces = {}
-        self.blacklist = blacklist
+        self.blacklister = bittensor.auth_blacklister()
```

Comment:
Is `bittensor.auth_blacklister` necessary inside the AuthInterceptor? If it is not needed, perhaps its better to only attach the blacklist function to keep the interceptor lightweight

Reply:
Agreed. Will change.